### PR TITLE
Experiments: Tweak - Indicators were missing tooltips that describes their state - ED-1193

### PR DESCRIPTION
**Pull request by rotemee**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: core/admin/admin.php [hunk 1]
```diff
@@ -72,6 +72,8 @@ public function maybe_redirect_to_getting_started() {
 	 * @access public
 	 */
 	public function enqueue_scripts() {
+		$suffix = ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG || defined( 'ELEMENTOR_TESTS' ) && ELEMENTOR_TESTS ) ? '' : '.min';
```
Comment (kobizz) [inline core/admin/admin.php:75]: Is it in use?

File: core/experiments/manager.php [hunk 2]
```diff
@@ -407,13 +407,15 @@ private function get_feature_settings_label_html( array $feature ) {
 		ob_start();
 
 		$indicator_classes = 'e-experiment__title__indicator';
+		$indicator_tooltip = __( 'Inactive by default', 'elementor' );
```
Comment (kobizz) [inline core/experiments/manager.php:410]: Use `Active` or `Inactive` when `$feature['state']` !== `self::STATE_DEFAULT`

**Review by kobizz — CHANGES_REQUESTED** [review]

File: core/experiments/manager.php (hunk 2 above)
Comment (rotemee) [reply core/experiments/manager.php]: Fixed.

File: core/admin/admin.php (hunk 1 above)
Comment (rotemee) [reply core/admin/admin.php]: No, was removed.

**Review by kobizz — DISMISSED** [review]

File: assets/dev/js/admin/admin.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import LandingPagesModule from 'elementor/modules/landing-pages/assets/js/admin/module';
+import ExperimentsModule from '../../../../core/experiments/assets/js/admin/module';
```
Comment (matipojo) [inline assets/dev/js/admin/admin.js:2]:
```suggestion
import ExperimentsModule from 'elementor/core/experiments/assets/js/admin/module';
```

File: core/experiments/assets/js/admin/module.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+			gravity: 's',
+			offset: 8,
+			title() {
+				return this.getAttribute( 'data-tooltip' );
+			},
+		} );
+	}
+
+	addIndicatorsTooltips() {
+		this.elements.$experimentIndicators.each( ( index, experimentIndicator ) => this.addTipsy( jQuery( experimentIndicator ) ) );
```
Comment (matipojo) [inline core/experiments/assets/js/admin/module.js:27]:
```suggestion
		this.elements.$experimentIndicators.each( ( index, $experimentIndicator ) => this.addTipsy( $experimentIndicator  ) );
```

**Review by matipojo — CHANGES_REQUESTED** [review]

File: core/experiments/assets/js/admin/module.js (hunk 4 above)
Comment (kobizz) [reply core/experiments/assets/js/admin/module.js:27]: The `experimentIndicator` is not a `jQuery` object

File: assets/dev/js/admin/admin.js (hunk 3 above)
Comment (rotemee) [reply assets/dev/js/admin/admin.js]: Fixed.

**Review by matipojo — DISMISSED** [review]

**Review by kobizz — DISMISSED** [review]

**Pull request merged**